src/types.ts

```diff
@@ -56,16 +56,16 @@ export interface FontVariant {
 
 export interface FontSetting {
   fontSize: FontSizeProperty<StyledValue>;
-  letterSpacing?: LetterSpacingProperty<StyledValue> | null;
-  lineHeight?: LineHeightProperty<StyledValue> | null;
+  letterSpacing?: LetterSpacingProperty<StyledValue>;
+  lineHeight?: LineHeightProperty<StyledValue>;
 }
 
 export interface StyledFont extends StyledObject {
   fontFamily: FontFamilyProperty;
   fontStyle?: FontStyleProperty;
   fontSize?: FontSizeProperty<StyledValue>;
-  fontWeight?: FontWeightProperty;
+  fontWeight?: FontWeightProperty | string;
```

Comment:
Curious as to why this is required.

Reply:
because if you use `"bold"` (like it is [here](https://github.com/buames/typesettings-js/blob/cad4332ed07bc8140d7b41c2cdc322666dc3d375/test/index.ts#L75)), a `string` is not a valid type. Which means that you would need the `csstype` package to import `FontWeightProperty` if you needed to type this, which is annoying.

Reply:
`FontWeightProperty` should handle this:

```
type FontWeightProperty = Globals | FontWeightAbsolute | "bolder" | "lighter";
type FontWeightAbsolute = "bold" | "normal" | number;
```

Reply:
It doesn't because `"bold"` and `"normal"` are types, not strings ...